src/transformers/models/deberta/modeling_tf_deberta.py

```diff
@@ -0,0 +1,1518 @@
+# coding=utf-8
+# Copyright 2021 kamalkraj and The HuggingFace Inc. team. All rights reserved.
```

Comment:
Hey @kamalkraj, the copyright here is generally reserved for the organisation that created the original model as well as HF for the library. The authorship of this PR remains as you're the author of the commit - and I invite you to add your name to this [documentation page](https://github.com/huggingface/transformers/blame/master/docs/source/model_doc/deberta.rst#L41-L42) as well to mention that you have added the TensorFlow version.

Reply:
updated and added name in documentaion page